source/epidemiology-other-compartmental-models.rst

```diff
@@ -56,6 +56,17 @@ If the agent is already infected, we transition them to the ``Removed`` compartm
 
  P_\text{IR} = \lambda_I \Delta t.
 
+Similarly to the SIR model, the average of the stochastic solutions should represent the mean field solution, as seen in the image below
+
+.. figure:: _static/images/epidemiology_SEIR_stochastic.png
+  :align: center
+  :alt: The stochastic solutions to the differential equations of the SEIR Model
+  :width: 600px
+  :figclass: align-center
+
+  $\lambda_S$=0.6/week, $\lambda_E$=0.5/week, $\lambda_I$=0.25/week
```

Comment:
Rates should be per day, not per week.

Reply:
I initially chose figures that'd be pretty simple in decimals per week. `lambda_E`, for instance, is 1/14 (once every two weeks). If you want me to express it per day, should I use a fraction (1/14 per day) over a decimal (~0.0714)?